src/char_mapping.rs

```diff
@@ -94,6 +131,51 @@ table! {
     }
 }
 
+table! {
+    #[doc = "A char-to-glyph encoding in format 12."]
+    pub Encoding12 {
+        format      (u16), // format
```

Comment:
Does it not make sense to have `= { 12 }` here and below?

Reply:
Indeed.